src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -156,6 +156,7 @@ public void writeThrough(User user) {
     userEntity.setProperty("username", user.getName());
     userEntity.setProperty("password_hash", user.getPasswordHash());
     userEntity.setProperty("creation_time", user.getCreationTime().toString());
+    userEntity.setProperty("about_me", user.getAboutMe());
```

Comment:
I see that you're saving the aboutMe info here, but I think you'll need to load it back from the store on line 70 above.